src/ZoneServer/Skills/Handlers/Swordsman/Doppelsoeldner/DeedsOfValor.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using Melia.Shared.L10N;
+using Melia.Shared.Game.Const;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+
+namespace Melia.Zone.Skills.Handlers.Swordsman.Doppelsoeldner
+{
+	/// <summary>
+	/// Handler for the Doppelsoeldner skill Deeds of Valor.
+	/// </summary>
+	[SkillHandler(SkillId.Doppelsoeldner_DeedsOfValor)]
+	public class DeedsOfValor : ISelfSkillHandler
+	{
+		public const float baseDamageMultiplier = 1.15f;  // 15% damage bonus
```

Comment:
These consts should be `PascalCase` as per our coding conventions.

Reply:
This has been fixed.